Components/UploadBox.tsx

```diff
@@ -72,7 +75,7 @@ const Upload = ({
                   <p className="mb-2 text-sm text-gray-500 dark:text-gray-400">
                     <span className="font-semibold">Click to upload</span>
                   </p>
-                  <p className="text-xs text-gray-500 dark:text-gray-400">.zip (max. 50MB)</p>
+                  <p className="text-xs text-gray-500 dark:text-gray-400">{`${fileType || '.zip'} `}(max. 50MB)</p>
```

Comment:
no need for a format-string here

```suggestion
                  <p className="text-xs text-gray-500 dark:text-gray-400">{fileType || '.zip'}(max. 50MB)</p>
```